packages/assets/styles/variables.scss

```diff
@@ -1,26 +1,110 @@
 .light-properties {
-	--color-bg: #ebebeb;
-	--color-raised-bg: #ffffff;
-	--color-super-raised-bg: #e9e9e9;
-	--color-button-bg: #f5f5f5;
+	--surface-l1: #ebebeb;
+	--surface-l2: #f5f5f5;
+	--surface-l3: #f8f8f8;
+	--surface-l4: #ffffff;
+	--surface-l5: #e6e6e6;
+
+	--color-red-50: #fef2f2;
+	--color-red-100: #fee5e7;
+	--color-red-200: #fccfd3;
+	--color-red-300: #faa7b1;
+	--color-red-400: #f67687;
+	--color-red-500: #ed4661;
+	--color-red-600: #cb2245;
+	--color-red-700: #b7193d;
+	--color-red-800: #9a1739;
+	--color-red-900: #841738;
+	--color-red-950: #490819;
+	--color-red: var(--color-red-600);
+
+	--color-orange-50: #fdf8ed;
+	--color-orange-100: #f9eacc;
+	--color-orange-200: #f2d495;
+	--color-orange-300: #ecb85d;
+	--color-orange-400: #e7a038;
+	--color-orange-500: #e08325;
+	--color-orange-600: #c66019;
+	--color-orange-700: #a44419;
+	--color-orange-800: #86351a;
+	--color-orange-900: #6e2d19;
+	--color-orange-950: #3f1509;
+	--color-orange: var(--color-orange-500);
+
+	--color-green-50: #eefff6;
+	--color-green-100: #d7ffeb;
+	--color-green-200: #b2ffd9;
+	--color-green-300: #76ffbc;
+	--color-green-400: #33f598;
+	--color-green-500: #09de78;
+	--color-green-600: #00af5c;
+	--color-green-700: #04914f;
+	--color-green-800: #0a7141;
+	--color-green-900: #0a5d38;
+	--color-green-950: #00341d;
+	--color-green: var(--color-green-600);
+
+	--color-blue-50: #f2f7fd;
+	--color-blue-100: #e4ecfa;
+	--color-blue-200: #c3d9f4;
+	--color-blue-300: #8ebaeb;
+	--color-blue-400: #5196df;
+	--color-blue-500: #2b79cc;
+	--color-blue-600: #1f68c0;
+	--color-blue-700: #184b8c;
+	--color-blue-800: #184174;
+	--color-blue-900: #193861;
+	--color-blue-950: #112340;
+	--color-blue: var(--color-blue-600);
+
+	--color-purple-50: #faf5ff;
+	--color-purple-100: #f2e7ff;
+	--color-purple-200: #e7d3ff;
+	--color-purple-300: #d4b1ff;
+	--color-purple-400: #ba7eff;
+	--color-purple-500: #9f4dff;
+	--color-purple-600: #8e32f3;
+	--color-purple-700: #761ad6;
+	--color-purple-800: #651bae;
+	--color-purple-900: #53178c;
+	--color-purple-950: #370368;
+	--color-purple: var(--color-purple-600);
+
+	--color-gray-50: #f5f5f6;
+	--color-gray-100: #e5e5e8;
+	--color-gray-200: #cecfd3;
+	--color-gray-300: #adb0b3;
+	--color-gray-400: #83868d;
+	--color-gray-500: #686a72;
+	--color-gray-600: #595b61;
+	--color-gray-700: #4c4e52;
+	--color-gray-800: #434447;
+	--color-gray-900: #3b3b3e;
+	--color-gray-950: #252627;
+	--color-gray: var(--color-gray-600);
+
+	--color-text-primary: #1a202c;
+	--color-text-default: #2c2e31;
```

Comment:
isnt this gonna be confusing as hell when in tailwind we call this `primary`?

Reply:
Yes, I decided not to migrate the existing text vars due to lack of time, in the future we can fix the aliases